chpl/chpl-resources/src/main/resources/email.properties

```
@@ -170,3 +170,8 @@ rwt_email_styles=table.blueTable {\
         padding: 2px 8px;\
         border-radius: 5px;\
     }
+
+# Real World Testing Report Email properties
+rwt.report.filename=real-world-testing-report-
+rwt.report.subject=ONC Real World Testing Report
```

Should this be:

```suggestion
rwt.report.subject=ONC-ACB Real World Testing Report
```

The ticket describes the report as an ONC-ACB report, so I'm not sure why this has "ONC" alone in the subject or body

---

Update to "CHPL Real World Testing Report" as per discussion in stand-up.